### PR TITLE
[wallet-ext] - update to wallet

### PR DESCRIPTION
**Pull request by mamos-mysten**
## Test Plan 

How did you test the new or updated feature?

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

**Comment by mamos-mysten** [comment]
Current dependencies on/for this PR:
* main
  * **PR #11485** <a href="https://app.graphite.dev/github/pr/MystenLabs/sui/11485" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/MystenLabs/sui/11485?utm_source=stack-comment).

File: apps/wallet/src/shared/experimentation/features.ts [hunk 1]
```diff
@@ -24,6 +24,7 @@ export const growthbook = new GrowthBook({
 export enum FEATURES {
     USE_LOCAL_TXN_SERIALIZER = 'use-local-txn-serializer',
     USE_TEST_NET_ENDPOINT = 'testnet-selection',
+    USE_MAIN_NET_ENDPOINT = 'mainnet-selection',
```
Comment (Jordan-Mysten) [inline apps/wallet/src/shared/experimentation/features.ts:27]: MAINNET not MAIN_NET

File: apps/wallet/configs/environment/.env.defaults [hunk 2]
```diff
@@ -1,11 +1,12 @@
 # Default values for some env variables. Use .env to add/change variables for the
 # current environment.
 
-API_ENV=testNet
+API_ENV=mainnet
```
Comment (Jordan-Mysten) [inline apps/wallet/configs/environment/.env.defaults:4]: Don't change the default

File: apps/explorer/src/components/network/Network.tsx [hunk 3]
```diff
@@ -1,26 +1,22 @@
...
 
 export default function WrappedNetworkSelect() {
     const [network, setNetwork] = useContext(NetworkContext);
     const { data } = useGetSystemState();
-    const showTestNet = useFeature(
-        GROWTHBOOK_FEATURES.USE_TEST_NET_ENDPOINT
-    ).on;
 
     const networks = [
+        { id: Network.MAINNET, label: 'Mainnet' },
```
Comment (Jordan-Mysten) [inline apps/explorer/src/components/network/Network.tsx:17]: Feature flag?

**Review by Jordan-Mysten — APPROVED** [review]

**Pull request merged**